promgen/templates/promgen/user_merge.html

```diff
@@ -0,0 +1,44 @@
+{% extends "base.html" %}
+{% load i18n static %}
+{% load admin_urls %}
+
+{% block title %}
+Promgen / Admin / Merge users
+{% endblock %}
+
+{% block content %}
+<div class="page-header">
+    <div>
+        <h1>Merge users</h1>
+    </div>
+</div>
+
+<ol class="breadcrumb" v-pre>
```

Comment:
```suggestion
<ol class="breadcrumb">
```

The Jinja code is not interpolating any variable that comes from a user input, therefore we don't need to protect this with `v-pre`.